templates/rhpam77-authoring-ha.yaml

```diff
@@ -1011,6 +1030,8 @@ objects:
             value: "${APPFORMER_JMS_BROKER_USER}"
           - name: APPFORMER_JMS_BROKER_PASSWORD
             value: "${APPFORMER_JMS_BROKER_PASSWORD}"
+          - name: JAVA_MAX_MEM_RATIO
```

Comment:
@errantepiphany Would it have sense to add this parameter to all Business central and Kie server deployments (templates and operator)? It may improve customer experience by having less out of memory issues while keeping same memory consumption (compared to the default settings).